integration-cli/docker_utils.go

```diff
@@ -463,6 +464,20 @@ func init() {
 			protectedImages[imgTag] = struct{}{}
 		}
 	}
+
+	// Obtain the daemon platform so that it can be used by tests to make
+	// intelligent decisions about how to configure themselves, and validate
+	// that the target platform is valid.
+	res, b, err := sockRequestRaw("GET", "/version", nil, "application/json")
+	defer b.Close()
+	if err != nil || res.StatusCode != http.StatusOK {
+		panic("Init failed to get version " + err.Error() + " " + string(res.StatusCode))
+	}
+	svrHeader, _ := httputils.ParseServerHeader(res.Header.Get("Server"))
+	daemonPlatform = svrHeader.OS
+	if daemonPlatform != "linux" && daemonPlatform != "windows" {
+		panic("Cannot run tests against platform " + daemonPlatform)
```

Comment:
very minor nit, but since we're not quoting the platform, can we add a ":" before it?
e.g. `...tests against platform: " + daemonPlatform)`